plugin.go

```diff
@@ -29,6 +30,7 @@ func (p *Plugin) Exec() error {
 	}
 
 	path := p.Path + p.Filename
+	fallback_path := p.FallbackPath + p.Filename
```

Comment:
Can we just make it `fallbackPath` I don't know if the snake case is following the Golang style